dataStandard_versioned_docs/version-5/whats-new/api_changes_5.md

```diff
@@ -0,0 +1,349 @@
+>   * Added property: shortenedSchoolDayIndicator (boolean)
+>   * Changed property: beginDate (string)
+>   * Changed property: endDate (string)
+>   * Changed property: iepBeginDate (string)
+>   * Changed property: iepEndDate (string)
+>   * Changed property: programParticipationStatuses (array)
+>   * Changed property: specialEducationProgramServices (array)
+>   * Changed property: programParticipationStatuses[n].statusBeginDate (string)
+>   * Changed property: programParticipationStatuses[n].statusEndDate (string)
+>   * Changed property: specialEducationProgramServices[n].serviceBeginDate (string)
+>   * Changed property: specialEducationProgramServices[n].serviceEndDate (string)
+### Time Variable Definition Updates
+The definition for the following time-related data elements have been updated.
+**/ed-fi/assessmentItems**
+expectedTimeAssessed //change into The duration allotted for the assessment item expressed in minutes.
+**/ed-fi/educationContents**
+timeRequired //change into Indicates if the total number of items available should be returned in the 'Total-Count' header of the response. If set to false, 'Total-Count' header will not be provided. Must be false when using cursor paging (with pageToken).
+**/ed-fi/studentInterventionAttendanceEvents**
+eventDuration //change into The amount of time in days for the event as recognized by the school: 1 day = 1, 1/2 day = 0.5, 1/3 day = 0.33.
+interventionDuration //change into The duration in minutes of the intervention attendance event.
+**/ed-fi/studentProgramEvaluations**
+evaluationDuration //change into The actual number of minutes to conduct the evaluation.
+**/ed-fi/surveyResponses**
+responseTime //change into The amount of time in seconds it took for the respondent to complete the survey.
+### Student Behavior and Discipline
+#### Add DisciplineIncident as an optional reference to restraintEvents
+* **GET** /ed-fi/restraintEvents
+##### Parameters
+>   * **Add** incidentIdentifier
+##### Response
+>   * Added property: [n].disciplineIncidentReference (object)
+* **POST** /ed-fi/restraintEvents
+##### Request
+>   * Added property: disciplineIncidentReference (object)
+* **GET** /ed-fi/restraintEvents/{id}
+##### Response
+>   * Added property: disciplineIncidentReference (object)
+* **PUT** /ed-fi/restraintEvents/{id}
+>   * Added property: disciplineIncidentReference (object)
+#### Add weapons descriptor to studentDisciplineIncidentBehaviorAssociations
+* **GET** /ed-fi/studentDisciplineIncidentBehaviorAssociations
+##### Response
+>   * Added property: [n].weapons (array)
+* **POST** /ed-fi/studentDisciplineIncidentBehaviorAssociations
+##### Request
+>   * Added property: weapons (array)
+* **GET** /ed-fi/studentDisciplineIncidentBehaviorAssociations/{id}
+##### Response
+>   * Added property: weapons (array)
+* **PUT** /ed-fi/studentDisciplineIncidentBehaviorAssociations/{id}
+>   * Added property: weapons (array)
+### Update definition for Begin and EndDate Data Elements
```

Comment:
I think we can delete the list of dates here and just say that we have added this note on all date-related data elements.